HousingSearchApi.Tests/V1/E2ETests/Stories/GetPersonsStories.cs

```diff
@@ -55,24 +55,6 @@ public void ServiceReturnsCorrectPageSize()
                 .BDDfy();
         }
 
-        [Fact]
-        public void ServiceReturnsCorrectAscSort()
-        {
-            this.Given(g => _personsFixture.GivenAPersonIndexExists())
-                .When(w => _steps.WhenARequestIsSortedByAFieldAsc())
-                .Then(t => _steps.ThenTheResultShouldBeSortedAsc())
-                .BDDfy();
-        }
-
-        [Fact]
-        public void ServiceReturnsCorrectDescSort()
-        {
-            this.Given(g => _personsFixture.GivenAPersonIndexExists())
-                .When(w => _steps.WhenARequestIsSortedByAFieldDesc())
-                .Then(t => _steps.ThenTheResultShouldBeSortedDesc())
-                .BDDfy();
-        }
-
```

Comment:
We don't use this feature anymore because it wasn't useful on MMH - no need to sort persons.

The new index mapping doesn't work with this because you need `keyword` fields to sort effectively